core/crypto.mk

```diff
@@ -228,3 +228,15 @@ _CFG_CORE_LTC_HASH := $(call ltc-one-enabled, MD5 SHA1 SHA224 SHA256 SHA384 \
 _CFG_CORE_LTC_MAC := $(call ltc-one-enabled, HMAC CMAC CBC_MAC)
 _CFG_CORE_LTC_CBC := $(call ltc-one-enabled, CBC CBC_MAC)
 _CFG_CORE_LTC_ASN1 := $(call ltc-one-enabled, RSA DSA ECC)
+
+###############################################################
+# Platform independent crypto-driver configuration
+###############################################################
+CRYPTO_MAKEFILES := $(shell find core/drivers/crypto -name crypto.mk)
```

Comment:
`$(wildcard core/drivers/crypto/*/crypto.mk)`

Reply:
Perhaps it should be sorted too, in order to have predictable builds.

Reply:
@jenswi-linaro agreed, [we've been bitten in the past](https://github.com/OP-TEE/build/commit/ea3ef223e506172bc27df2bc793e4bb3d82c9408) ;-)

So that would be `CRYPTO_MAKEFILES := $(sort $(wildcard core/drivers/crypto/*/crypto.mk))`.

Reply:
pushed update